tasks/run-provision.yaml

```diff
@@ -21,6 +23,32 @@
     file: check-run-tower-job.yaml
 
 - name: Schedule check for provision of {{ anarchy_subject_name }}
+  when: >-
+    not __meta__.deployer.actions.provision.disable | default(false) | bool
   anarchy_continue_action:
     after: "{{ job_check_interval }}"
+
+- name: Handle action complete for {{ anarchy_subject_name }} when deployer is disabled
+  when: >-
+    sandbox_api_in_use | bool and
+    __meta__.deployer.actions.provision.disable | default(false) | bool
+  block:
+  - name: Set state started for {{ anarchy_subject_name }} when deployer is disabled
+    vars:
+      action_provision_data: "{{ extracted_sandboxes_vars | default({}) }}"
+    anarchy_subject_update:
+      metadata:
+        labels:
+          state: started
+      spec:
+        vars:
+          current_state: started
+          healthy: true
+          provision_data: '{{ action_provision_data | default(omit, true) }}'
+          provision_message_body: '{{ action_provision_message_body | default(omit, true) }}'
+          provision_messages: '{{ action_provision_messages | default(omit, true) }}'
```

Comment:
There would never be messages or message body, right?

Reply:
right